app/Components/Sudoku Board/SudokuBoard.tsx

```diff
@@ -438,13 +440,69 @@ const SudokuBoard = (props: SudokuBoardProps) => {
     const inputValue = event.nativeEvent.key;
     if (/^[1-9]$/.test(inputValue)) {
       updateCellEntry(parseInt(inputValue, 10));
-    }
-    if (
+    } else if (
       inputValue == "Delete" ||
       inputValue == "Backspace" ||
-      inputValue == "0"
-    )
+      inputValue == "0" ||
+      inputValue == "e" ||
+      inputValue == "E" // e and E are for erase
+    ) {
       eraseSelected();
+    } else if (inputValue == "u" || inputValue == "U") {
+      undo();
+    } else if (inputValue == "p" || inputValue == "P") {
+      saveGame(sudokuBoard);
+      navigation.goBack();
```

Comment:
Would it be possible to reuse the handlePause function located in HeaderRow.tsx?

Reply:
Good idea, done!